security/openconnect/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		openconnect
-PLUGIN_VERSION=		1.3.3
+PLUGIN_VERSION=		1.3.4
```

Comment:
I would prefer 1.4 due to a mimimal API change.

In semantic versioning the revision (3rd number means only bugfixes, second means minimal changes which may require adjusting scripts etc. while a major is maybe totally incompatible or not backward compatible).

Reply:
Indeed, small new feature, I'll bump to 1.4.0